src/hooks/userspace/hook-grpc.js

```diff
@@ -144,8 +144,6 @@ function sendMetadataWrapper(rootContext) {
     return function sendMetadataTrace(responseMetadata) {
       if (rootContext) {
         rootContext.addLabel('metadata', JSON.stringify(responseMetadata.getMap()));
-      } else {
-        agent.logger.info('gRPC: No root context found in sendMetadata');
```

Comment:
If this is the case, the if statement above should be removed too and a brief comment added.